qt/res/app.organicmaps.desktop.metainfo.xml

```diff
@@ -107,6 +107,14 @@
   </screenshots>
 
   <releases>
+   <release version="2022.05.31-10" date="2022-05-31">
+     <description>
+       <ul>
+         <li>Do not draw place=region.</li>
+         <li>Fix crash on routing.</li>
```

Comment:
I took the liberty of adding this since it's a pretty important fix for flatpak

Reply:
Did we fix the crash?

Reply:
According to https://github.com/flathub/app.organicmaps.desktop/issues/2 the crash was fixed in https://github.com/organicmaps/organicmaps/pull/2625, which is included in 2022.05.31-10: https://github.com/organicmaps/organicmaps/commit/1a631163449fbecd0182c5f4a57123b63bf48bda

![image](https://user-images.githubusercontent.com/8182846/171683406-b2689b78-fc6c-42d0-a0ad-f470bc5079ea.png)

Reply:
Good, sorry I forgot about it :)